packages/ra-ui-materialui/src/list/SimpleList.tsx

```diff
@@ -271,35 +280,52 @@ const LinkOrNot = (props: LinkOrNotProps) => {
         id,
         children,
         record,
+        resource,
     } = props;
     const classes = useLinkOrNotStyles({ classes: classesOverride });
-    const link =
-        typeof linkType === 'function' ? linkType(record, id) : linkType;
+    const [effect, setEffect] = useState<string>(() =>
+        linkType === 'edit' || linkType === true
```

Comment:
if linktype was false, there was no link. Your change breaks that.

Reply:
Indeed, I'll fix it